packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "f6581f2e8fcb07074fca9e72b3134476f6fc74ab",
-    "ref_origin" : "dcos-mesos-1.0.1"
+    "ref": "b546e7544c04a6d3e0da87cd6207ae8f4d827add",
+    "ref_origin" : "dcos-mesos-1.0.1-logrotate"
```

Comment:
Let's cherry-pick these onto `dcos-mesos-1.0.1` so we have a clear linear progression of patches added onto one branch off an official Apache Mesos release (1.0.1), rather than potentially losing patches between the many "feature" branches. Totally appropriate for testing, but I'm convinced that it's time to cherry-pick these onto the "official" branch now.